integration/environment/instance_group.go

```diff
@@ -10,14 +11,18 @@ import (
 
 // WaitForInstanceGroup blocks until all selected pods of the instance group are running. It fails after the timeout.
 func (m *Machine) WaitForInstanceGroup(namespace string, deployment string, igName string, version string, count int) error {
+	return m.WaitForInstanceGroupMultiVersions(namespace, deployment, igName, count, version)
+}
+
+// WaitForInstanceGroupMultiVersions blocks until all selected pods of the instance group are running. It fails after the timeout.
+func (m *Machine) WaitForInstanceGroupMultiVersions(namespace string, deployment string, igName string, count int, versions ...string) error {
```

Comment:
Should this be `WaitForInstanceGroupGenerationAllReplicas` (based on the comment)?
Also, are we calling it `Version` or `Generation`?

Reply:
We renamed the method.

With regards to `version` vs. `generation` - we would like to stay with `version` because the `generation` used in kubernetes is not exactly the same. Especially changing the `partition` will increase the `generation`.